### PR TITLE
[IMP] sign: secured authentication info added

### PR DESCRIPTION
**Pull request by Donapi**
[ID: 3084552](https://www.odoo.com/web#id=3084552&cids=1&model=project.task&view_type=form)

### CODE REVIEW

File: content/applications/finance/sign.rst [hunk 1]
```diff
@@ -92,12 +92,73 @@ sign it.
...
+Go to :menuselection:`Sign --> Configuration --> Roles`. Click in the
+:guilabel:`Extra Authentication Step` column for the role, and select :guilabel:`Unique Code Via
+SMS`.
+
+.. image:: sign/sms-verification.png
+   :align: center
+   :alt: Add a hash to your document
+
+Go to the document to sign, add the :guilabel:`Signature` field for example, and click
+:guilabel:`Send`. A page pops up, select the customer, and click :guilabel:`Send`.
```
Comment (LoredanaLrpz) [inline content/applications/finance/sign.rst:124]: Not sure about "previous", but here's what I'd suggest
```suggestion
Go to the document to sign and add a field that has the previous role attributed, such as the
:guilabel:`Signature` field, and click :guilabel:`Send`. A page pops up, select the customer, and
click :guilabel:`Send`.
```

File: content/applications/finance/sign.rst [hunk 2]
```diff
@@ -92,12 +92,73 @@ sign it.
...
+.. note::
+   - This feature is enabled by default.
+   - As soon as the :guilabel:`Extra Authentication Step` applies to a role, this validation step is
+     requested for any field assigned to this role.
+
+Itsme®
+~~~~~~
+
+Go to :menuselection:`Sign --> Configuration --> Settings` and enable
+:guilabel:`Identify with itsme®`.
```
Comment (LoredanaLrpz) [inline content/applications/finance/sign.rst:143]:
```suggestion
Go to :menuselection:`Sign --> Configuration --> Settings` and enable :guilabel:`Identify with
itsme®`.
```

File: content/applications/finance/sign.rst [hunk 3]
```diff
@@ -92,12 +92,73 @@ sign it.
...
+
+Then, go to :menuselection:`Sign --> Configuration --> Roles`. Click in the
+:guilabel:`Extra Authentication Step` column for the role, and select :guilabel:`Via itsme®`.
```
Comment (LoredanaLrpz) [inline content/applications/finance/sign.rst:146]:
```suggestion
Then, go to :menuselection:`Sign --> Configuration --> Roles`. Click in the :guilabel:`Extra
Authentication Step` column for the role, and select :guilabel:`Via itsme®`.
```

File: content/applications/finance/sign.rst [hunk 4]
```diff
@@ -92,12 +92,73 @@ sign it.
...
+
+Go to the document that needs to be signed, add the :guilabel:`Signature` field, and select the role
+:guilabel:`customer (identified with itsme®)`, and click :guilabel:`Validate` and :guilabel:`Send`.
```
Comment (LoredanaLrpz) [inline content/applications/finance/sign.rst:149]: That sentence contains a lot of "and" here's what I'd suggest:
```suggestion
Go to the document that needs to be signed and add the :guilabel:`Signature` field. Switch the role
to :guilabel:`customer (identified with itsme®)`, click :guilabel:`Validate`, and :guilabel:`Send`.
```

File: content/applications/finance/sign.rst [hunk 5]
```diff
@@ -92,12 +92,73 @@ sign it.
...
+
+.. image:: sign/itsme-identification.png
+   :align: center
+   :alt: select customer identified with itsme®
+
+The person signing the document fills in the :guilabel:`Signature` field and then click
```
Comment (LoredanaLrpz) [inline content/applications/finance/sign.rst:155]:
```suggestion
The person signing the document fills in the :guilabel:`Signature` field and clicks
```

**Review by LoredanaLrpz — CHANGES_REQUESTED** [review]
hello @Donapi , there you have it :)

**Review by LoredanaLrpz — COMMENTED** [review]
Hey @Donapi , the PR is great, I'll approve it as soon as the pictures issue is fixed =)
Have a nice one!

**Review by LoredanaLrpz — APPROVED** [review]

**Review by AntoineVDV — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**